Add details about configuring g/w HA

aswinsuryan opened the pull request:
Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

aswinsuryan committed:
Add details about configuring g/w HA

Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

nyechiel commented on src/resources/shared/openshift/run_prep_for_subm.md:
````diff
@@ -13,3 +13,10 @@ export IPSEC_NATT_PORT=4501
 export GW_INSTANCE_TYPE=m4.xlarge
 ```
+
+Submariner gateway can deployed in HA mode by setting the ENABLE_HA flag.
````
```suggestion
Submariner gateway can be deployed in HA mode by setting the ENABLE_HA flag:
```

nyechiel commented on src/content/getting-started/quickstart/openshift/vsphere-aws/_index.md:
````diff
@@ -118,6 +118,12 @@ export IPSEC_NATT_PORT=4501
 export IPSEC_IKE_PORT=501
 ```
 
+* To deploy the gateway in HA mode set the ENABLE_HA flag
````
```suggestion
* To deploy the gateway in HA mode set the ENABLE_HA flag:
```

nyechiel reviewed (commented):
Thanks @aswinsuryan. Can you please also update the in-repo README file? https://github.com/submariner-io/submariner/tree/devel/tools/openshift/ocp-ipi-aws

Also, there is a Markdown linting failure that needs to be addressed.

aswinsuryan committed:
Apply suggestions from code review

Co-authored-by: Nir Yechiel <n.yechiel@gmail.com>
Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

aswinsuryan commented:
> Thanks @aswinsuryan. Can you please also update the in-repo README file? https://github.com/submariner-io/submariner/tree/devel/tools/openshift/ocp-ipi-aws
> 
> Also, there is a Markdown linting failure that needs to be addressed.

I have addressed the comments, will raise a PR for README.
